### PR TITLE
LS-40 Setup deployment

### PR DESCRIPTION
**Pull request by hkevgill**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -238,7 +238,7 @@ module.exports = function (grunt) {
         src: [
           '<%= yeoman.dist %>/scripts/{,*/}*.js',
           '<%= yeoman.dist %>/styles/{,*/}*.css',
-          '<%= yeoman.dist %>/images/{,*/}*.{png,jpg,jpeg,gif,webp,svg}',
```
Comment (hkevgill) [inline Gruntfile.js]: Grunt attempts to rename the images for better browser caching. But there is a bug in grunt that doesn't properly rename the image when it is used in html (works fine when used in css). So the workaround until grunt gets this issue fixed is to just not rename our images when our app is built

File: Gruntfile.js [hunk 2]
```diff
@@ -336,7 +336,7 @@ module.exports = function (grunt) {
         files: [{
           expand: true,
           cwd: '<%= yeoman.dist %>',
-          src: ['*.html', 'views/{,*/}*.html'],
```
Comment (hkevgill) [inline Gruntfile.js]: Just forgot to initially change this.

File: Gruntfile.js [hunk 3]
```diff
@@ -374,7 +374,7 @@ module.exports = function (grunt) {
             '*.{ico,png,txt}',
             '.htaccess',
             '*.html',
-            'views/{,*/}*.html',
```
Comment (hkevgill) [inline Gruntfile.js]: And this.

**Comment by nfeliciano** [comment]
+1

**Comment by chrisjcook** [comment]
:+1:

**Pull request merged**